### ISSUE TITLE
Add JakartaOne Livestream Banner to Jakarta.ee

### ISSUE THREAD

**Issue by hudsonkelly83** [jakartaee/jakarta.ee#898]
Please add the following to jakarta.ee banner & footer rotation

Date: Sept 23(or whenever you get to it) - Dec 8 
Link for register button: https://jakartaone.org/2020/
Link for Download Button: needs to be linked to file of jakarta ee flags hosted on jakarta.ee -
![Jakarta9 Flag](https://user-images.githubusercontent.com/70344546/93893303-7c236600-fcbb-11ea-8136-f032dc51b653.png)

Banner File: 
![image](https://user-images.githubusercontent.com/70344546/93893669-e5a37480-fcbb-11ea-8310-901062a28228.png)

Footer File: 
![image](https://user-images.githubusercontent.com/70344546/93893754-0075e900-fcbc-11ea-8e81-54616188c0a2.png)

Please let me know if there are any questions.

**Comment by hudsonkelly83** [comment]
Please also remove the CNJ Banner from rotation (pictured below)
![image](https://user-images.githubusercontent.com/70344546/94180056-6d7bb100-fe6b-11ea-8aec-ad96c69c9e76.png)

**Comment by flora8984461** [comment]
Hello Kelly, 
There are some I am not quite sure. 

- Do you want me to add footer above `Sign up to our Newsletter` or after? 
- What is the blue image with words "JakartaEE 9" for? Is this the flag to be downloaded?
- What do you mean by rotation? 

Thanks for your clarification!

**Comment by hudsonkelly83** [comment]
No problem, 

1) If you could add the footer right above the 'Sign up to our Newsletter' that would be perfect

2) The blue image is a flag yes, please excuse my limited graphic design abilities. We would like for people to be able to download this image by clicking the 'DOWNLOAD' button, so it will have to live somewhere on the jakarta.ee website for that to happen, I'm guessing. I created this issue based on a previous one basically requesting the same thing, here it is: https://github.com/jakartaee/jakarta.ee/issues/807 

3) By rotation I just mean add it to the group of banners that currently display on Jakarta EE's website, for example when I open the page this is what i see for a banner: 
![image](https://user-images.githubusercontent.com/70344546/94184613-293fdf00-fe72-11ea-96aa-c903509db70b.png)

but if i refresh, it changes to: 
![image](https://user-images.githubusercontent.com/70344546/94184657-3c52af00-fe72-11ea-97e2-6169c671022d.png)
Basically, we just need to swap out the JakartaOne Livestream Cloud Native for Java, to the new banner, JakartaOne Livestream 2020. 

I hope this helps, if you need more just let me know

**Comment by autumnfound** [comment]
@flora8984461 the "Sign up for newsletter" section is the featured footer, and you shouldn't add a new section or duplicate instance of the component.

In terms of rotation, this refers to adding it to the featuredstory.yml file as a new entry with a valid date. When there are multiple valid entries, they "rotate" on reload. In actuality, it's fully random and we serve a random valid footer/story on page load through JS. 

For the flag, we would download the image, place it in the ./static/images/ folder, and link to it for the download link. When we have events, we like to provide assets for users to download as a way of promoting the event when/if they post images to social media. This lets them show pride in the platform and help us get the good word of Jakarta out to more people!

**Comment by flora8984461** [comment]
As @autumnfound mentioned, I cannot directly add another footer above the "Sign up for newsletter"  footer, so I add it to be rotationally loaded like the banner.
